documentation/source/install/cloud_solutions.rst

```diff
@@ -257,76 +257,94 @@ Minimum requirements for container cluster are as follows:
 
 The example commands below were run on a container cluster *demo-openebs03* in zone *us-central1-a* with project unique ID *strong-eon-153112*. When you copy paste the command, ensure that you use the details from your project.
 
-You can either use an existing container cluster or create a new one. To create a new cluster, go to **Google Cloud Platform** -> **Container Engine** -> **Create Container Cluster** for example, *demo-openebs03*. 
+iSCSI Configuration
+^^^^^^^^^^^^^^^^^^^^^
 
-Add iSCSI Support
-------------------
-
-Go to **Google Cloud Platform** -> **Compute Engine** -> **VM instances** to install open-iscsi package. OpenEBS uses iSCSI to connect to the block volumes. The nodes displayed are Compute Engine VMs, and you can see them in the console. The display is similar to the following screen.
+Go to **Google Cloud Platform** -> **Compute Engine** -> **VM instances**. The nodes displayed by default in this console are Compute Engine VMs, and you can see them in the console. The display is similar to the following screen.
  
  .. image:: ../_static/compute_engine_vms.png
 
+**Verify that iSCSI is configured**
 
-Select one of the SSH nodes displayed in the cluster, click **SSH**, and run the following commands.
-::
-
-    sudo apt-get update
-    sudo apt-get install open-iscsi
-    sudo service open-iscsi restart
-
-Verify that iSCSI is configured
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-1. Check that initiator name is configured and iSCSI service is running using the following commands.
+a. Check that initiator name is configured.
 ::
 
     sudo cat /etc/iscsi/initiatorname.iscsi
```

Comment:
sudo cat /etc/iscsi/initiatorname.iscsi = ~$ sudo cat /etc/iscsi/initiatorname.iscsi

To differentiate between command and output.

Reply:
Done!